client/public/config/configuration.json

```diff
@@ -19,6 +19,9 @@
     "DEV_EXISTING_NR_TIMEOUT_MS": 0,
     "PAYMENT_COMPLETION_TIMER_NAME": "paymentCompletionTimer",
     "PAYMENT_COMPLETION_TIMEOUT_MS": 120000,
-    "DEV_PAYMENT_COMPLETION_TIMEOUT_MS": 900000
+    "DEV_PAYMENT_COMPLETION_TIMEOUT_MS": 900000,
+    "AUTH_TOKEN_URL": "https://dev.oidc.gov.bc.ca/auth/realms/fcf0kpqr/protocol/openid-connect/token",
+    "QUICK_SEARCH_PUBLIC_ID": "",
+    "QUICK_SEARCH_PUBLIC_SECRET": ""
```

Comment:
Don't forget to add these to the config maps in Dev/Test/Prod. (You could probably do it now as it won't break anything.)

Reply:
doing it now